### ISSUE TITLE
eval 

### ISSUE THREAD

**Issue by gtgtgt1117** [google/automl#256]
@Jupanlee 

 Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.206
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.330
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.231
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.332
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = -1.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = -1.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.324
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = 0.426
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.427
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.427
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = -1.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = -1.000

请问我的验证结果正常吗?

**Comment by Jupanlee** [comment]
你可以加入微信群和大家交流呢，这样会更及时一些

gtgtgt1117 <notifications@github.com> 于2020年4月17日周五 下午4:42写道：

**Comment by 15241192958** [comment]
> 群和大家交

请问微信群是哪个？能方便分享下吗

**Comment by elesun2018** [comment]
when i run eval code on coco with pretrained models, meet a problem as follows:
automl0401/efficientdet# python main.py --mode=eval --model_name=${MODEL} --model_dir=${CKPT_PATH} --validation_file_pattern=tfrecord/val* --val_json_file=annotations/instances_val2017.json --hparams="use_bfloat16=false" --use_tpu=False

CKPT_PATH is the pretrained d0 model.

DONE (t=34.86s).
Average Precision (AP) @[ IoU=0.50:0.95 | area= all | maxDets=100 ] = 0.002
Average Precision (AP) @[ IoU=0.50 | area= all | maxDets=100 ] = 0.005
Average Precision (AP) @[ IoU=0.75 | area= all | maxDets=100 ] = 0.002
Average Precision (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.003
Average Precision (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.001
Average Precision (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.000
Average Recall (AR) @[ IoU=0.50:0.95 | area= all | maxDets= 1 ] = 0.006
Average Recall (AR) @[ IoU=0.50:0.95 | area= all | maxDets= 10 ] = 0.011
Average Recall (AR) @[ IoU=0.50:0.95 | area= all | maxDets=100 ] = 0.012
Average Recall (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.018
Average Recall (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.006
Average Recall (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.004
INFO:tensorflow:Finished evaluation at 2020-04-28-17:12:04
INFO:tensorflow:Finished evaluation at 2020-04-28-17:12:04
I0428 17:12:04.599111 140494972888896 evaluation.py:275] Finished evaluation at 2020-04-28-17:12:04
INFO:tensorflow:Saving dict for global step 0: AP = 0.0022378154, AP50 = 0.0052162167, AP75 = 0.0015809378, APl = 0.00015473206, APm = 0.0014778649, APs = 0.0029822278, ARl = 0.0036031725, ARm = 0.006258475, ARmax1 = 0.005900326, ARmax10 = 0.011014862, ARmax100 = 0.011780873, ARs = 0.018089531, box_loss = 0.0, cls_loss = 0.02997985, global_step = 0, loss = 0.11981841
I0428 17:12:04.600037 140494972888896 estimator.py:2049] Saving dict for global step 0: AP = 0.0022378154, AP50 = 0.0052162167, AP75 = 0.0015809378, APl = 0.00015473206, APm = 0.0014778649, APs = 0.0029822278, ARl = 0.0036031725, ARm = 0.006258475, ARmax1 = 0.005900326, ARmax10 = 0.011014862, ARmax100 = 0.011780873, ARs = 0.018089531, box_loss = 0.0, cls_loss = 0.02997985, global_step = 0, loss = 0.11981841
INFO:tensorflow:Saving 'checkpoint_path' summary for global step 0: models/pretrained_models/efficientdet-d1/model

such low AP values ? why ?
thanks.
